winlogbeat/beat/winlogbeat.go

```diff
@@ -94,7 +95,8 @@ func (eb *Winlogbeat) Setup(b *beat.Beat) error {
 	eb.done = make(chan struct{})
 
 	var err error
-	eb.checkpoint, err = checkpoint.NewCheckpoint(eb.config.Winlogbeat.RegistryFile, 10, 5*time.Second)
+	eb.checkpoint, err = checkpoint.NewCheckpoint(
+		eb.config.Winlogbeat.RegistryFile, 10, 5*time.Second)
```

Comment:
I don't think the the time interval *needs* to be configurable, but it *might* be useful to make it configurable (so a separate PR). With fewer knobs to turn, I'm hope there are fewer questions about how to configure the beat.